user_data/strategies/test_strategy.py

```diff
@@ -11,11 +14,10 @@
 
 
 # Update this variable if you change the class name
-class_name = 'TestStrategy'
+class_name = 'DefaultStrategy'
 
 
-# This class is a sample. Feel free to customize it.
```

Comment:
Any particular reason to remove the explanation that this is a sample class only?

Reply:
No reason, I can add it back. I'm going to try and make a revision that is debugged here in a minute and re-edit this PR  :)